e2e/acceptance_test.go

```
@@ -0,0 +1,319 @@
+	// Postgres in CI would let every TestAcc* skip and the job would go green
+	// having verified nothing.
+	dsn := os.Getenv("SQL_DSN")
+	if dsn == "" {
+		t.Skip("SQL_DSN not set; skipping acceptance test")
+	}
+
+	db, err := sql.Open("postgres", dsn)
+	if err != nil {
+		t.Fatalf("open SQL_DSN: %v", err)
+	}
+	defer db.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)